client/components/marketing-survey/cancel-purchase-form/stepsForProductAndSurvey.js

```diff
@@ -20,17 +20,12 @@ const JETPACK_PAID_PLANS = [
 	plans.PLAN_JETPACK_PREMIUM_MONTHLY,
 ];
 
-export default function stepsForProductAndSurvey( survey, product, canChat ) {
-	if ( survey && survey.questionOneRadio === 'tooHard' ) {
-		if ( includesProduct( BUSINESS_PLANS, product ) ) {
-			return [ steps.INITIAL_STEP, steps.CONCIERGE_STEP, steps.FINAL_STEP ];
-		}
-
-		if ( canChat && includesProduct( PERSONAL_PREMIUM_PLANS, product ) ) {
-			return [ steps.INITIAL_STEP, steps.HAPPYCHAT_STEP, steps.FINAL_STEP ];
-		}
-	}
-
+export default function stepsForProductAndSurvey(
+	survey,
+	product,
+	canChat,
+	precancellation = false
```

Comment:
For me, it isn't quite clear why `precancellation` would have default argument as `false` while the others don't. In my opinion, default arguments should only used when it's the most common sensical value. Could we consider to remove this and fill in the call site with proper value?

Reply:
Just to be clear, I remove the default argument in the function declaration so now we have this:

```js
export default function stepsForProductAndSurvey(
	// ...
	precancellation,
) {
	// ...
}
```

Now, wherever I call the function, I'm providing a default value during the destructuring:

``` js

const { precancellationChatAvailable = false } = this.props;

const steps = stepsForProductAndSurvey(
	// ...
	precancellationChatAvailable,
);
```

Just wanted to make sure that's what you're referring to!